verify

```diff
@@ -124,7 +124,14 @@ function verify_rpm() {
 
 	# install all non-source packages
 	(
-		set -x
+		set -ux
```

Comment:
`set -u` should be added to the _whole_ script (not just this one bit) but it's got a decent chance of breaking stuff, so I'd leave it off completely for now and consider adding it later (I just wanted to make sure this bit of script was ready when you're ready to think about that :innocent:)

Reply:
Ah, yes, I didn't want to jinx it by putting it for the whole script 😅